Sources/RuleBuilder/Property.swift

```diff
@@ -37,3 +37,37 @@ public struct Property: Text {
         }
     }
 }
+
+
+public extension Property {
+    /// https://bazel.build/concepts/visibility
+    public enum Visibility {
+        /// "//visibility:public": Anyone can use this target. (May not be combined with any other specification.)
+        public static let `public`: Property = "visibility" => "//visibility:public"
+        /// "//visibility:private": Only targets in this package can use this target. (May not be combined with any other specification.)
+        public static let `private`: Property = "visibility" => "//visibility:private"
```

Comment:
This new operator is readable. But not good for jumping to definition.

Reply:
We can add Readme according to [PropertyTests](https://github.com/XCodeBazelize/Bazelize/blob/new_builder/Tests/RuleBuilderTests/PropertyTests.swift).